`Development`: Add documentation for standardized competencies (#8711)
ls1intum/Artemis: docs/user/adaptive-learning/adaptive-learning-admin.rst

Manage Standardized Competencies (Administrator)
==================================================

.. contents:: Content of this document
    :local:
    :depth: 2

.. _standardized_competency_catalog:

Standardized Competencies
-------------------------

| The **Standardized Competency Catalog** is a catalog of competencies that is accessible to all instructors in an Artemis instance.
  Instructors may view this catalog and import competencies from it into their course.

> | The competencies consist of a title, description and taxonomy. Instead of being part of courses, they are sorted under knowledge areas.

Reviewer: ```suggestion
| The competencies consist of a title, description and taxonomy. Instead of being tied to specific courses, competencies are organized under knowledge areas.
```

| Knowledge areas may also contain other knowledge areas, enabling a hierarchical structure (e.g., Artificial Intelligence -> Machine Learning).

Manage Standardized Competencies
--------------------------------

.. raw:: html

    <iframe src="https://live.rbg.tum.de/w/artemisintro/46942?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
        Watch this video on TUM-Live.
    </iframe>

| Administrators can manage the standardized competency catalog of their Artemis instance
  in the *Standardized Competency Management* view.

> | They can filter the displayed content by by competency title and knowledge area.

Reviewer: ```suggestion
| They can filter the displayed content by competency title and knowledge area.
```

  Clicking on a knowledge area expands it and clicking on the eye icon shows the knowledge area details on the right.

> | From this detail view they can edit or delete the knowledge area, or create new knowledge areas and competencies under it.

Reviewer 2: ```suggestion
| From this details view they can edit or delete the knowledge area, or create new knowledge areas and competencies under it.
```
